apps/condo/domains/billing/utils/validation.utils.js

```diff
@@ -22,7 +22,7 @@ const PAYMENT_SCHEMA = {
 
 const _jsonPaymentObjectSchemaValidator = ajv.compile(PAYMENT_SCHEMA)
 
-function _validatePaymentDetails ({ resolvedData, fieldPath, addFieldValidationError }) {
+function validatePaymentDetails ({ resolvedData, fieldPath, addFieldValidationError }) {
     if (!_jsonPaymentObjectSchemaValidator(resolvedData[fieldPath])) {
```

Comment:
We haven't naming rules for private members, consider rename it to `jsonPaymentObjectSchemaValidator` and use camelCase